backend/api/src/main/kotlin/software/shonk/application/service/ShorkService.kt

```diff
@@ -70,6 +69,28 @@ class ShorkService(private val shork: IShork) : ShorkUseCase {
         return Result.success(lobby.getStatus())
     }
 
+    override fun getAllLobbies(): List<LobbyStatus> {
+        val lobbiesList = mutableListOf<LobbyStatus>()
+
+        for (lobbyId in lobbies.keys) {
+            getLobbyStatus(lobbyId).getOrNull()?.let { status ->
+                lobbiesList.add(
+                    LobbyStatus(
+                        lobbyId = lobbyId,
+                        playersJoined =
+                            listOfNotNull(
+                                    if (status.playerASubmitted) "playerA" else null,
+                                    if (status.playerBSubmitted) "playerB" else null,
+                                )
+                                .ifEmpty { listOf("no player submission yet") },
```

Comment:
there was no mention of what an empty list of no player submissions could give back in the spec. Thus, I improvised a simple message for that

Reply:
Yes. The spec does not go out of its way to specify what to return instead of an empty list, as the empty list itself should be specified.
`"no player submission yet"` would be treated as a player ID by the callee, as that is what the spec specifies.